beacon-chain/rpc/eth/beacon/handlers_test.go

```diff
@@ -4754,3 +4755,213 @@ func Test_validateBlobSidecars(t *testing.T) {
 	require.NoError(t, err)
 	require.ErrorContains(t, "could not verify blob proof: can't verify opening proof", s.validateBlobSidecars(b, [][]byte{blob[:]}, [][]byte{proof[:]}))
 }
+
+func TestGetPendingDeposits(t *testing.T) {
+	config := params.BeaconConfig()
+	st, _ := util.DeterministicGenesisStateElectra(t, config.MaxValidatorsPerCommittee)
```

Comment:
Using `config.MaxValidatorsPerCommittee` for the numbers of validators in a state is a bit unusual. Also we want to use a small number because this function is very involved. Since you are creating 20 pending deposits later, you don't need more validators than that.